api/nakadi-event-bus-api.yaml

```diff
@@ -1246,7 +1258,15 @@ definitions:
     properties:
       name:
         type: string
-        description: Name of the strategy.
+        enum: ['hash', 'user_defined', 'random']
+        description: |
+          Name of the strategy. Possible values are:
+          - 'hash' will use the event field(s) defined in 'partitioning_key_fields' property of `EventType` as a source
+            for a hash function that will caclulate the partition where the event will be put.
```

Comment:
Maybe make it even more clear that "all events with the same value in this field will go to the same partition, and consequently be ordered".